_gsocproposals/2022/proposal_ATHENA-Clustering-on-GPUs.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Electromagnetic Cluster Finding on GPUs
+layout: gsoc_proposal
+project: ATHENA
+year: 2022
+difficulty: high
+duration: 350
+mentor_avail: June-August, September-October
+organization: EIC
+---
+## Description
+
+The ATHENA collaboration at the Electron-Ion Collider (EIC) is planning to build
+a novel electromagnetic calorimeter (ECAL) that combines AstroPix silicon pixel
+detector imaging layers with plastic scintillating fibers embedded in lead and
+read out at the ends. Due to the increase in segmentation that is associated with
+the 500-um pitch pixel detectors, the computational demands on finding clusters
+of hits caused by a single particle shower are much larger than for traditional
+electromagnetic calorimeters. For typical events that we expect to encounter at
+the EIC, we expect to spend the majority of computing time on this cluster
+reconstruction if we use traditional algorithms. Since the EIC experiments will
+use streaming readout, this corresponds to a bottleneck on the data rates that we
+will be able to sustain.
+
+In this project we aim to parallelize the clustering algorithm to run efficiently
+on GPUs, and separate the clustering algorithms into a separate library with an
+interface based on the standardized EDM4hep data model. We aim to focus from the 
+start on the SYCL cross-platform abstraction layer, mirroring an approach which 
+is already used by other components of our event reconstruction algorithm.
+
+## Technology:
+ * C/C++, template programming, GPU programming.
+
+## Desirable Skills
+ * Necessary knowledge: C, Java or C++ programming; data structures and algorithms.
+ * Experience with GPU programming or numerical analysis would be an asset.
+
+## Mentors
+ * [Wouter Deconinck](mailto:wouter.deconinck@umanitoba.ca) (University of Manitoba)
```

Comment:
```suggestion
 * **[Wouter Deconinck](mailto:wouter.deconinck@umanitoba.ca)** (University of Manitoba)
```

Reply:
Please make sure you add Argonne and Manitoba to the _gocorgs folder . Pleaase add the mentor names alphabetically in 2022/mentors.md. I was planning to do it myself, but there are more necessary changes.